variables.tf

```diff
@@ -143,6 +143,16 @@ variable "gratis_dns" {
   default = null
 }
 
+variable "route_53_dns" {
```

Comment:
Please rename from `route_53_dns` to `route53`. (This is a conversion rule from camel/pascal case to snake_case.)